packages/backend/src/scheduler/SchedulerTask.ts

```diff
@@ -896,6 +903,27 @@ export default class SchedulerTask {
                     organizationUuid: user.organizationUuid,
                 });
             }
+            const canReplaceCustomMetrics = await isFeatureFlagEnabled(
+                FeatureFlags.ReplaceCustomMetricsOnCompile,
+                {
+                    userUuid: user.userUuid,
+                    organizationUuid: user.organizationUuid,
+                },
+                {
+                    // because we are checking this in the health check, we don't want to throw an error
```

Comment:
we might need to update this comment, this does not happen on healthcheck

Reply:
removed comment and timeout 👍